packages/foreman/foreman-selinux/foreman-selinux.spec

```diff
@@ -68,12 +64,7 @@ SELinux policy module for Foreman
 
 %build
 # determine distribution name and version
-%if 0%{?rhel} >= 6
 %define distver rhel%{rhel}
```

Comment:
Not a blocker for this, but it may not be a bad idea to enhance foreman-selinux to determine this automatically.

Reply:
Technically, since https://github.com/theforeman/foreman-selinux/commit/0c2d1f0972fca43a008d20260f5983bef4d6ed5e, we don't need to define that at all, as we don't have any distro specific macros.

Reply:
When I looked at that I wondered if the shared make rules would use it. However, that appears to use distro in a different way:
```console
$ rg DISTRO /usr/share/selinux/devel
/usr/share/selinux/devel/include/Makefile
56:ifneq ($(DISTRO),)
57:	M4PARAM += -D distro_$(DISTRO)

/usr/share/selinux/devel/include/build.conf
3:DISTRO ?= redhat
```

So I'm leaning to dropping it: https://github.com/theforeman/foreman-selinux/pull/150

Reply:
@evgeni shall we merge that PR and then update this PR?

Reply:
I think that's a plan!